velox/vector/fuzzer/VectorFuzzer.cpp

```diff
@@ -239,17 +239,46 @@ void fuzzFlatPrimitiveImpl(
   }
 }
 
+// Servers as a wrapper around a vector that will be used to load a lazyVector.
+// Ensures that the loaded vector will only contain valid rows for the row set
```

Comment:
what do we do for the other rows? Add some garbage?

Reply:
Makes sense. We had a discussion in the past that these uninitialized values could be set to garbage in the fuzzer to ensure that functions don't rely on them being zero or other properly initialized values. But this is probably outside of the scope of this PR.

Reply:
I had thought about this earlier but came to the following conclusion: 
It might be difficult to come up with a garbage value that would illicit an error from a piece of code that reads rows which it should not. As an example, a null buffer can either be true or false (which is always valid), a scalar value can have any value that is valid. Moreover it might be difficult to add garbage values that would let us create valid vectors, eg. a dictionary index needs to have a value between 0 and dictionarySize otherwise it will hit a Dcheck in the constructor.
However, if we are comparing the result against a non lazy vector input which we will do in the expression fuzzer (after my next patch is merged) when comparing with the result from Simple eval, we should see a mismatch in results with a high probability and catch the bug